spec/livefeed/parser_spec.rb

```diff
@@ -306,6 +306,10 @@ module Twingly::LiveFeed
       context "with an unauthorized api key result" do
         let(:fixture) { :unauthorized_api_key }
 
+        it "should raise AuthenticationError" do
+          expect { subject }.to raise_error(AuthenticationError)
+        end
+
         it "should raise AuthError" do
```

Comment:
I think we can remove this now. The spec was added above, and the spec added in `error_spec.rb` should be enough.

Reply:
Yeah, true. A bit too much overlap.

Reply:
71ce125